src/modules/mavlink/mavlink_high_latency2.h

```diff
@@ -44,6 +44,12 @@
 #include "mavlink_simple_analyzer.h"
 #include "mavlink_stream.h"
 
+class SimplerAnalyzer : public SimpleAnalyzer
+{
+public:
+	SimplerAnalyzer() : SimpleAnalyzer(SimpleAnalyzer::AVERAGE) {}
```

Comment:
Why don't you make `SimpleAnalyzer::AVERAGE` the default for `SimpleAnalyzer`?
Or you collect all per-num-brick members into a struct with appropriate constructor:
```
struct PerBatteryData {
  PerBatteryData() : battery(SimpleAnalyzer::AVERAGE) {}
  MavlinkOrbSubscription *battery_sub;
  SimpleAnalyzer battery;
  ...
};
```